src/Main/IntroContainer.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import BigLogo from './BigLogo'
+import Intro from './Intro'
+
+const IntroContainer = (props) => {
+  return (
+  	<div className="intro-container">
+  		<Intro />
+    	<BigLogo />
```

Comment:
Distance between big logo and intro container should be bigger. Perhaps we can also adjust the margins of the entire page container to be smaller than they are now?